src/Components/Button/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactNode, FC, MouseEventHandler } from "react";
+
+import { Container, ContainerProps } from "./style";
+
+interface ButtonProps extends ContainerProps {
+  children: ReactNode;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
```

Comment:
```suggestion
interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
  children: ReactNode;
```

요렇게 button element의 attributes를 확장하면 어떨까요?

Reply:
넵! 이름은 onClick으로 해놓고 html attribute type을 사용할 생각은 못했네요 ㅎㅎ
수정했습니다!
commit link : https://github.com/woowacourse/react-shopping-cart/pull/12/commits/5296a7df06a0a79a2d55d7f2db764c0049a424fd

Reply:
사실 children도 명시 안 해줘도 됩니당 ㅎㅎ